sklearn/feature_selection/_univariate_selection.py

```diff
@@ -237,7 +237,7 @@ def r_regression(X, y, *, center=True, force_finite=True):
     a free standing feature selection procedure.
 
     The cross correlation between each regressor and the target is computed
-    as ((X[:, i] - mean(X[:, i])) * (y - mean_y)) / (std(X[:, i]) * std(y)).
+    as E[(X[:, i] - mean(X[:, i])) * (y - mean(y))] / (std(X[:, i]) * std(y)).
```

Comment:
I think we can improve the rendering here:

```python
    The cross correlation between each regressor and the target is computed as::

        E[(X[:, i] - mean(X[:, i])) * (y - mean(y))] / (std(X[:, i]) * std(y))
```

#### Before


![Screen Shot 2022-01-25 at 3 55 30 PM](https://user-images.githubusercontent.com/5402633/151058239-62b261a7-9f18-4e78-95f4-10b29dd3b870.jpg)

#### After

![Screen Shot 2022-01-25 at 3 55 56 PM](https://user-images.githubusercontent.com/5402633/151058296-3ef61f50-513f-464d-a995-25f0633dc2d2.jpg)